client/bottom-slot.js

```diff
@@ -12,52 +12,52 @@ const BANNER_LINK_SELECTOR = `.${BANNER_CLASS}__link`;
 const BOTTOM_SLOT_FLAG = 'messageSlotBottom';
 
 module.exports = function ({ config={}, guruResult, customSetup }={}) {
-	let banner;
 	const guruRenderData = guruResult && guruResult.renderData || {};
-	const variant = (guruRenderData && guruRenderData.dynamicTrackingData) || config.name;
+	const trackEventAction = getTracker(config, guruRenderData);
 
-	const trackingContext = Object.assign(
-		config.trackingContext || {},
-		guruRenderData && guruRenderData.trackingContext || {},
-	);
-
-	const trackEventAction = config.name && generateMessageEvent({
-		flag: BOTTOM_SLOT_FLAG,
-		messageId: config.name,
-		position: config.slot,
-		trackingContext,
-		variant
-	});
+	const skip = guruResult && guruResult.skip;
+	const bannerLimitBreached = messageEventLimitsBreached(config.name);
+	if (skip | bannerLimitBreached) {
```

Comment:
```suggestion
	if (skip || bannerLimitBreached) {
```

replace bitwise OR with logical OR 😬 I think in this case the result is the same but whoops